packages/desktop-client/src/components/common/Tooltip.tsx

```diff
@@ -52,7 +52,7 @@ export const Tooltip = ({
 
   return (
     <View
-      style={{ minHeight: 'auto' }}
+      style={{ minHeight: 'auto', flexShrink: 0 }}
```

Comment:
<img width="596" alt="Screenshot 2024-10-11 at 19 47 09" src="https://github.com/user-attachments/assets/f8bd0aa6-83ee-4308-bb80-2dc09487e18f">

fix for the tiny icons